src/Cinelovers/Controls/LoadingIndicator.xaml.cs

```diff
@@ -0,0 +1,50 @@
+﻿using ReactiveUI;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Cinelovers.Controls
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class LoadingIndicator : ContentView
```

Comment:
This implementation is fine and even has a well done animation on the text, but it is not very pretty. Do you have any suggestions on how to improve it?

Reply:
Yes, I chose this custom one just to be simple and maintain consistency between all platforms since UWP doesn't have any indicator built in within its `ListView` as iOS and Android have. 

I can see some options to resolve this:
1: Use the `ListView` built-in indicator for iOS/Android, and for UWP use an `ActivityIndicator` instead.
2: Use an ActivityIndicator for all of them.
3: Use a third party plugin, maybe `Acr.UserDialogs` or another that could help. Actually in most of the projects I decided for this approach but here I prefer to not involve a third party for this simple task.